demo/src/main/java/com/jira/demo/repository/TaskRepo.java

```diff
@@ -3,5 +3,6 @@
 
 import com.jira.demo.model.Task;
 import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
Don't forget to format your code with combination of `Ctrl + Alt + L` to keep your code clean and consistent